engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -105,6 +110,11 @@ paths:
           required: true
           type: file
           in: formData
+        - name: workflowOnHold
+          description: Put workflow on hold upon submission?
+          required: false
```

Comment:
Does swagger allow us to say what the default is (presumably `false`?)

Reply:
FWIW I've had bad luck w/ defaults & enums in swagger, so if this route is tried make sure that it actually works as expected

Reply:
I tried adding a default value, but it doesn't work. I also tried using enum along with default but that doesn't work too. So I have added comment in description of parameter (in Swagger UI) saying that default behavior is false.